site/gatsby-site/src/contexts/userContext/UserContext.js

```diff
@@ -5,7 +5,8 @@ export const UserContext = createContext({
   user: undefined,
   isAdmin: undefined,
   isLoggedIn: undefined,
-  isRole() {
+  isRole(role) {
+    role;
```

Comment:
this is weird but I get why you did it 🤷

Reply:
It's to avoid eslint errors 🤷